hydrolib/core/io/rr/topology/serializer.py

```diff
@@ -0,0 +1,70 @@
+import os
+from pathlib import Path
+
+
+class NodeFileSerializer:
```

Comment:
More of a side note than something you really have to adjust now, but you could separate these to two separate files, and then just make regular unbounded functions, instead of grouping them in classes. Though I think in this particular case it makes sense to group them.

Reply:
I'm still trying to development my senses in regards to "python" structuring. But thanks, will keep it in mind!